### ISSUE TITLE
Block comments need a unique pattern for easier parsing

### ISSUE THREAD

**Issue by kristoferjoseph** [enhance-dev/enhance-styles#19]
Proposal is to update
```
/* THEME */
```
To something more unique like:
```
/* --- THEME --- */
```
Or the like so that it is easier to discern from other block comments.

While updating these we should make sure that the header comments make sense for the logical groupings we want to show in the generated cheat sheet.

**Comment by colepeters** [comment]
What are all the cases we need to cover with these comments? You mentioned other block comments and logical groupings — just trying to wrap my head around everything we need to account for here.

**Comment by ryanbethel** [comment]
This is probably clear from my hacky regex, but just in case. When I did a quick version of the cheatsheet that categorized by headings in comments the hardest regex conditions to work around were:
- ignoring the inline or end of line comments that look similar to the headings.
- ignoring some headings that you don't want to show up in the cheatsheet. I had to explicitly filter some of these out after the results were generated.
If you avoid this confusion by making them distinct in the generated output, like has been mentioned in this thread already, it will minimize the regex shenanigans.
